alica_engine/src/engine/modelmanagement/ModelManager.cpp

```diff
@@ -120,53 +86,53 @@ Plan* ModelManager::loadPlanTree(const std::string& masterPlanName)
 RoleSet* ModelManager::loadRoleSet(const std::string& roleSetName)
 {
     std::string roleSetPath;
```

Comment:
```suggestion
    autro roleSetPath = findFile(roleSetName + alica::Strings::roleset_extension);
```
and you don't need such long expression inside `if`